Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -173,12 +170,11 @@ void EmbeddedFrag::computeNbrsAndAng(unsigned int aid,
   //  inside the ring We want to find ang(BAC) instead - which we will this do
   //  by checking that both our neighbors are not involved in more than one
   //  ring. Bridged systems - don't even go there
-  DOUBLE_INT_PAIR winner = anglePairs.back();
-  for (auto apcri = anglePairs.rbegin(); apcri != anglePairs.rend(); ++apcri) {
-    const auto &nbrPair = apcri->second;
-    if ((dp_mol->getRingInfo()->numAtomRings(nbrPair.first) <= 1) &&
-        (dp_mol->getRingInfo()->numAtomRings(nbrPair.second) <= 1)) {
-      winner = (*apcri);
+  auto winner = anglePairs.back();
+  for (auto pr : boost::adaptors::reverse(anglePairs)) {
```

Comment:
```suggestion
  for (const auto &pr : boost::adaptors::reverse(anglePairs)) {
```

Reply:
same argument about pair being small and cheap-to-copy